web_src/less/_base.less

```diff
@@ -1977,14 +1977,6 @@ table th[data-sortt-desc] {
   margin-right: 0 !important;
 }
 
-/* limit width of all direct dropdown menu children */
-/* https://github.com/go-gitea/gitea/pull/10835 */
-.dropdown:not(.selection) > .menu:not(.review-box) > *:not(.header) {
-  max-width: 300px;
-  overflow-x: hidden;
-  text-overflow: ellipsis;
-}
-
```

Comment:
I guess some reviewers may ask "why" here. So I answer ahead: these code doesn't make sense, and the problem described by #10835 doesn't exist at the moment (as least for my testing).

There should be no global pollution for `max-width`, there are also more issues related to it, in which users complain about "why the width of the dropdown is so narrow".